systemtests/tests/restapi/api/restapi.sh

```diff
@@ -34,12 +34,14 @@ else
     export PYTHONPATH=${CMAKE_SOURCE_DIR}/restapi/
 fi
 
+LSOF_CMD="lsof -n -iTCP:$REST_API_PORT"
```

Comment:
```suggestion
LSOF_CMD="lsof -n -iTCP:$REST_API_PORT -sTCP:LISTEN"
```

Shouldn't we add `-sTCP:LISTEN` here?
I think that only the listening state is relevant here, and looks like `lsof` doesn't distinguish between source and target port.